.azure-pipelines/templates/variables.yml

```diff
@@ -1,4 +1,4 @@
 variables:
-  ubuntu_pool: 'pool-ubuntu-2004'
+  ubuntu_pool: 'pool-ubuntu-2204'
```

Comment:
The reason for not using `pool-ubuntu-latest` is that Azure PowerShell team has already created `pool-ubuntu-2204`.